js/src/ui/Features/store.js

```diff
@@ -0,0 +1,73 @@
+// Copyright 2015, 2016 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+
+// Parity is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity.  If not, see <http://www.gnu.org/licenses/>.
+
+import { action, observable } from 'mobx';
+import store from 'store';
+
+import { LS_STORE_KEY } from './constants';
+import defaults, { FEATURES, MODES } from './defaults';
+
+const isProductionMode = process.env.NODE_ENV === 'production';
+
+let instance = null;
+
+export default class Store {
+  @observable active = {};
+
+  constructor () {
+    this.loadActiveFeatures();
+  }
+
+  @action setActiveFeatures = (features = {}, isProduction) => {
+    this.active = Object.assign({}, features, this.getDefaultActive(isProduction));
```

Comment:
There was method to the madness, what it was I don't recall - it has been a bit too long between writing this. However, now that dev mode is toggle-able, it would make sense.